.readme-partials.yaml

```diff
@@ -359,9 +361,12 @@ body: |-
   - `$POOL_ID`: The workload identity pool ID.
   - `$AWS_PROVIDER_ID`: The AWS provider ID.
   - `$SERVICE_ACCOUNT_EMAIL`: The email of the service account to impersonate.
-
+  - `$TOKEN_LIFETIME`: The desired lifetime duration of the service account access token in seconds.
+  
   This will generate the configuration file in the specified output file.
 
+  The `service-account-token-lifetime-seconds` flag is optional. If not provided, this defaults to one hour. If a lifetime greater than one hour is required, the service account must be added as an allowed value in an Organization Policy that enforces the `constraints/iam.allowServiceAccountCredentialLifetimeExtension` constraint.
```

Comment:
We should document the min/max values.

Also document that setting the service account token lifetime to a smaller value (e.g. 10 minutes) will result in the lib initiating the entire token exchange flow every 10 minutes (and we will call the 3PI token sourcing method every time, even if the 3PI token is not expired).